recipes/statsmodels/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "statsmodels" %}
+{% set version = "0.6.1" %}
+{% set build = 0 %}
+{% set md5 = "f7580ebf7d2a2c9b87abfad190dcb9a3" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    md5: {{ md5 }}
+
+build:
+    number: {{ build }}
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - cython
+        - numpy x.x
+        - scipy
+        - pandas
+        - patsy
+    run:
+        - python
+        - numpy x.x
+        - scipy
+        - pandas
+        - patsy
+test:
+    imports:
+        - statsmodels
+
+about:
+    home: https://www.statsmodels.org
+    license: BSD 3
```

Comment:
Fixed. This would be a good addition to the linter :) (Looks like Jan suggested it here https://github.com/conda-forge/conda-smithy/issues/184)